landing/src/lib/components/NeuralTopology.svelte

```
@@ -561,90 +934,172 @@
 
     raf = requestAnimationFrame(render);
   });
 
   onDestroy(() => {
     if (typeof cancelAnimationFrame !== "undefined") cancelAnimationFrame(raf);
-    if (typeof window !== "undefined")
+    if (typeof window !== "undefined") {
       window.removeEventListener("resize", resize);
+      if (onMouseMove) window.removeEventListener("mousemove", onMouseMove);
+    }
     sectionObservers.forEach((io) => io.disconnect());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)